docs/reference/commandline/network_create.md

```diff
@@ -14,18 +14,111 @@ parent = "smn_cli"
+    -o --opt=map[]           Set custom network plugin options
+    --subnet=[]              Subnet in CIDR format that represents a network segment
 
-Creates a new network that containers can connect to. If the driver supports multi-host networking, the created network will be made available across all the hosts in the cluster. Daemon will do its best to identify network name conflicts. But its the users responsibility to make sure network name is unique across the cluster. You create a network and then configure the container to use it, for example:
+Creates a new network. The `DRIVER` accepts `bridge` or `overlay` which are the built-in network drivers. If you have installed a third party or your own custom network driver you can specify that `DRIVER` here also. If you don't specify the `--driver` option, the command automatically creates a `bridge` network for you.
 
+When you install Docker Engine it creates a `bridge` network automatically.
+This network corresponds to the `docker0` bridge that Engine has traditionally
+relied on. When you `run` a new container it automatically launches on this
+bridge network. You cannot remove this default bridge network but you can create
+new ones using the `network create` command.
+
+```bash
+$ docker network create -d bridge my-bridge-network
+```
+
+Bridge networks are isolated networks on a single Engine installation. If you
+want to create a network that spans multiple Docker hosts each running an
+Engine, you must create an `overlay` network. Unlike `bridge` networks overlay
+networks require some pre-existing conditions before you can create one. These
+conditions are:
+
+* Access to a key-value store. Engine supports Consul Etcd Zookeeper (Distributed store) and BoltDB (Local store) key-value stores.
+* A cluster of hosts with connectivity to the key-value store.
+* A properly configured Engine `daemon` on each host in the cluster.
+
+The `docker daemon` options that support the `overlay` network are:
+
+* `--cluster-store`
+* `--cluster-store-opt`
+* `--cluster-advertise`
+
+It is also a good idea, though not required, that you install Docker Swarm on
+to manage the cluster. Swarm provides sophisticated discovery and server
+management that can assist your implementation. Once you have prepared the
+`overlay` network prerequisites you simply choose a Docker host in the cluster
+and issue the following to create the network:
+
+```bash
+$ docker network create -d overlay my-multihost-network
 ```
-  $ docker network create -d overlay multi-host-network
-  $ docker run -itd --net=multi-host-network busybox
+
+Network names must be unique. The Docker daemon attempts to identify naming
+conflicts but this is not guaranteed. It is the user's responsibility to avoid
+name conflicts.
+
+## Connect containers
+
+When you start a container use the `--net` flag to connect it to a network.
+This adds the `busybox` container to the `mynet` network.
```

Comment:
Not in this case.  I don't want to use the same network in every example or it gives the impression that they can only do a mhs network.